src/collectors/httpflat/httpflat.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+
+"""
+Simple collector which get FLAT files and parse it into flat metrics
+
+#### Dependencies
+
+ * urllib2
+
+"""
+
+import urllib2
+import diamond.collector
+
+
+class HTTPFLATCollector(diamond.collector.Collector):
+
+    def get_default_config_help(self):
+        config_help = super(HTTPFLATCollector, self).get_default_config_help()
+        config_help.update({
+            'url': 'Full URL'
+        })
+        return config_help
+
+    def get_default_config(self):
+        default_config = super(HTTPFLATCollector, self).get_default_config()
+        default_config.update({
+            'path': 'httpflat',
+            'url': 'http://localhost/stat'
+        })
+        return default_config
+
+    def _flat_to_flat_metrics(self, prefix, data):
```

Comment:
This whole thing should just be done as a generator. yielding for each line. Right now, you're splitting the lines upfront, then yielding all the parsed values. Would be better to iterate each line and parse separately.